newapi/scan_error_handlers.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+"""
+Extract error handlers from ooniapi
+
+jer("error code", "error formatter string", a=1, b=2, ...)
+
+Error out if the handlers are different from error_handlers.json
+"""
```

Comment:
It is neat that we're importing `ast` to solve this problem. However, this solution brings with it some extra mounts of complexity, such as parsing the `ast`. I would like to know more about the design of this feature and what other alternative designs that could solve the problem were rejected.